packages/cli/src/ActiveWebhooks.ts

```diff
@@ -6,8 +6,9 @@ import type {
 	WorkflowActivateMode,
 	WorkflowExecuteMode,
 } from 'n8n-workflow';
-import { ApplicationError, WebhookPathTakenError } from 'n8n-workflow';
+import { WebhookPathTakenError } from 'n8n-workflow';
 import * as NodeExecuteFunctions from 'n8n-core';
+import { IdLessWorkflowError } from './errors/id-less-workflow.error';
 
 @Service()
 export class ActiveWebhooks {
```

Comment:
while we are at it, can you please move all of `ActiveWebhooks` into `TestWebhooks` and delete this file?
I'd like to create another `ActiveWebhooks` at some point to break out al webhooks code from `ActiveWorkflowRunner`, but that can wait for now.